.flowconfig

```diff
@@ -31,6 +31,7 @@ module.name_mapper='^rewards\(.*\)$' -> '<PROJECT_ROOT>/ui/rewards\1'
 module.name_mapper='^i18n\(.*\)$' -> '<PROJECT_ROOT>/ui/i18n\1'
```

Comment:
I get the nag from my country with Bahasa Melayu as the national language.
- [ ] There is no homepage for my language, yet I can choose "Both" and "Select Homepage".  Does it make sense?
- [ ] When I select "Select Language", I expected "Bahasa Melayu" to be selected, but I got Chinese selections instead.
    - <img width="363" alt="image" src="https://user-images.githubusercontent.com/64950861/155728519-b89e8446-d6be-4f8d-bf98-efbf16ba4d66.png">

Reply:
It's my fault cause I had too quickly read some stuff that chinese is spoken in MY, and asked him to override. Should remove that override @saltrafael, sorry.

Reply:
Ah, I see the code now.  I think Singapore's default (`sg: 'en,ms,zh,ta'`) make sense for both MY and SG, where and English and Malay is the two main languages, and Chinese and Tamil are the secondary ones.
Probably need to undo the override for SG too since it's English-centric.